bot/utils/scheduling.py

```diff
@@ -59,14 +59,18 @@ def schedule(self, task_id: t.Hashable, coroutine: t.Coroutine) -> None:
 
     def schedule_at(self, time: datetime, task_id: t.Hashable, coroutine: t.Coroutine) -> None:
         """
-        Schedule `coroutine` to be executed at the given naïve UTC `time`.
+        Schedule `coroutine` to be executed at the given `time`.
+
+        If `time` is timezone aware, then use that timezone to calculate now() when subtracting.
+        If `time` is naïve, then we use UTC.
```

Comment:
First-person (and second-person) pronouns should be avoided, especially in docstrings. Use the imperative mood.

```suggestion
        If `time` is naïve, then use UTC.
```